XamlControlsGallery/ConnectedAnimationPages/CollectionPage.xaml.cs

```diff
@@ -16,7 +16,7 @@ public CollectionPage()
         {
             this.InitializeComponent();
 
-            // Ensure that the MainPage is only created once, and cached during navigation.
+            // Ensure that the MainPage is only created                                          once, and cached during navigation.
```

Comment:
I think one space is enough :)
```suggestion
            // Ensure that the MainPage is only created once, and cached during navigation.
```